### PR TITLE
remove unused and misspelling word hashd

### PR DESCRIPTION
**Pull request by javaTyrant**
## What is the purpose of the change

remove unused field

## Brief changelog

remove unused field

## Verifying this change

No logical changes

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-common/src/main/java/org/apache/dubbo/common/extension/factory/AdaptiveExtensionFactory.java [hunk 1]
```diff
@@ -34,22 +33,19 @@
...
-            if (extension != null) {
-                return extension;
-            }
-        }
-        return null;
+        return factories.stream()
+                .filter(Objects::nonNull)
+                .findFirst()
+                .map(extensionFactory -> extensionFactory.getExtension(type, name))
+                .orElse(null);
```
Comment (chickenlj) [inline dubbo-common/src/main/java/org/apache/dubbo/common/extension/factory/AdaptiveExtensionFactory.java:49]: This is not the same meaning of the original one.

File: dubbo-common/src/main/java/org/apache/dubbo/common/extension/factory/AdaptiveExtensionFactory.java [hunk 2]
```diff
@@ -20,9 +20,8 @@
 import org.apache.dubbo.common.extension.ExtensionFactory;
 import org.apache.dubbo.common.extension.ExtensionLoader;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
```
Comment (chickenlj) [inline dubbo-common/src/main/java/org/apache/dubbo/common/extension/factory/AdaptiveExtensionFactory.java:23]: list every import explicitly.

**Comment by chickenlj** [comment]
Please reopen when fixed.

File: dubbo-common/src/main/java/org/apache/dubbo/common/extension/factory/AdaptiveExtensionFactory.java (hunk 2 above)
Comment (javaTyrant) [reply dubbo-common/src/main/java/org/apache/dubbo/common/extension/factory/AdaptiveExtensionFactory.java:23]: okay

**Pull request closed without merging**